.circleci/config.yml

```diff
@@ -18,8 +18,66 @@ jobs:
           PATH=$HOME/miniconda3/envs/pandas-dev/bin:$HOME/miniconda3/condabin:$PATH
           LD_PRELOAD=$HOME/miniconda3/envs/pandas-dev/lib/libgomp.so.1:$LD_PRELOAD
           ci/run_tests.sh
+  build-aarch64:
+    parameters:
+      cibw-build:
+        type: string
+    machine:
+      image: ubuntu-2004:202101-01
+    resource_class: arm.large
+    environment:
+      TAG = << pipeline.git.tag >>
+      TRIGGER_SOURCE = << pipeline.trigger_source >>
+    steps:
+      - checkout

```

Comment:
Not a big deal, but is there a way to limit the fetch depth?

Reply:
The checkout step can't be configured. We could use raw git commands, but I don't think its worth the time savings (we still need to pull tags).